src/rezplugins/shell/cmd.py

```diff
@@ -239,13 +243,68 @@ def escape_string(self, value, is_path=False):
             else:
                 if is_path:
                     txt = self.normalize_paths(txt)
+                elif is_shell_path:
+                    txt = self.as_shell_path(txt)
 
                 txt = self._escaper(txt)
             result += txt
         return result
 
+    def as_path(self, path):
+        """
+        Return the given path as a system path.
+        Used if the path needs to be reformatted to suit a specific case.
+        Args:
+            path (str): File path.
+
+        Returns:
+            (str): Transformed file path.
+        """
+        # Prevent path conversion if normalization is disabled in the config.
+        if config.disable_normalization:
```

Comment:
Could this be done in the normalize_path path instead? I'm not 100% convinced it should, so just asking to get different opinions.